hooks/sentry_hook.py

```diff
@@ -87,13 +87,19 @@ def add_sentry(task_instance, *args, session=None, **kwargs):
             capture_exception()
             raise
 
+def get_dsn(conn):
+    if None in (conn.conn_type, conn.login):
+        return conn.host
+
+    dsn = f"{conn.conn_type}://{conn.login}@{conn.host}/{conn.schema}"
```

Comment:
love the `f` syntax but we may want to favor the traditional `.format()` pattern instead of compatibility with python2.